engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html

```diff
@@ -28,6 +28,15 @@
     </button>
 
     <button class="btn btn-default"
+            ui-sref="products.details.repositories.manage-ostree-branches({productId: product.id, repositoryId: repository.id})"
+            ng-hide="repository.content_type !== 'ostree' || product.readonly">
+      <span translate>Manage Branches</span>
+    </button>
+
+
+
+
```

Comment:
I thought you practiced Zen mediation where one tries to keep a "blank state" :)